CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## v2.0.0-rc.1 / 2017-10-17
+
+* [BUGFIX] Fixed Remote-read to not drop the first series.
+* [BUGFIX] Validate recording-rule names.
+* [BUGFIX] Fix several races.
+* [BUGFIX] Only close blocks if there are no iterators accessing it.
+* [FEATURE] Added a warning for time-drift between the browser and the prometheus-server.
```

Comment:
Our usual order is: change, feature, enhancement, bugfix.

Reply:
I've done this repeatedly as well, how about we document this in the "HOWTO cut a new release" document?

Reply:
Be my guest :-)

Reply:
Done :)

Reply:
Fixed and will keep in mind from next time.